include/TPP/Passes.td

```diff
@@ -256,6 +247,16 @@ def RewriteBatchMatmulToMatmul : Pass<"rewrite-batch-matmul-to-matmul",
   let dependentDialects = ["scf::SCFDialect", "linalg::LinalgDialect"];
 }
 
+def CombineXsmmOpPass : Pass<"combine-xsmm-op-optimization", "func::FuncOp"> {
+  let summary = "Fuse gemm-relu-bias into a bigger xsmm op";
+  let description =
+      [{Fuse gemm-relu-bias into a bigger xsmm op }];
+  let constructor = "mlir::tpp::createCombineXsmmOpPass()";
```

Comment:
nit: from now on each new pass should not have `let constructor` this is generated by tablegen. You can have a look here on how to retire it: https://github.com/chelini/tpp-sandbox/commits/fix-vnni2

Reply:
fixed